cmd/agentbasedinstaller/register.go

```diff
@@ -192,7 +192,7 @@ func getReleaseVersionAndCpuArch(log *log.Logger, releaseImage string, releaseMi
 		return "", "", archError
 	}
 
-	return version, cpuArch, nil
+	return version, cpuArch[0], nil
```

Comment:
is it intentional that we always return the 1st cpuArch here?
Is there a concept of a "default" index or some way for the user to pick a different index?

Reply:
@zaneb that's your part of the code that does not have a coverage anywhere in `cmd/agentbasedinstaller/*test.go`; can you have a look please? The semantics of `GetReleaseArchitecture()` changes to return `[]string` instead of `string`. I tried to accommodate your function as much as I understood it, but given there are no other handlers apart for error returned by `GetReleaseArchitecture()`, we end up with such a simple change